webextensions/api/tabs.json

```diff
@@ -905,7 +924,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": false
+                  "version_added": "121"
```

Comment:
FYI, added by: https://issues.chromium.org/issues/40895236
https://chromium.googlesource.com/chromium/src/+/8a56b5cf35a863f2fe0c0089c80df4546ecb7814
confirmed to be version 121 per https://chromium.googlesource.com/chromium/src/+/8a56b5cf35a863f2fe0c0089c80df4546ecb7814/chrome/VERSION

Reply:
@Rob--W so are you suggesting we shouldn't rely solely on the Chrome documentation?

Reply:
You can continue to rely on Chrome's doc. I was adding a source to confirm that the feature has indeed landed and is available there. Inaccuracies in Chrome's documentation has happened before, and since this was only one change I decided to double-check.